libs/pages/elewa/home/src/lib/features-elewa-home.module.ts

```diff
@@ -3,10 +3,24 @@ import { CommonModule } from '@angular/common';
 import { HomePageComponent } from './pages/home-page/home-page.component';
 import { HomeHeroSectionComponent } from './components/home-hero-section/home-hero-section.component';
 import { LayoutModule } from '@elewa-group/elements/layout';
+import { JobsSectionComponent } from './components/jobs-section/jobs-section.component';
+import { Routes, RouterModule } from '@angular/router';
+
+// routes
+const routes: Routes = [
+  {
+    path: 'jobs',
+    component: JobsSectionComponent,
+  },
+];
 
```

Comment:
routes are done on a routing file, not here.
Just put the route url on the html, proper routing will be done in a separate issue.